polaris-icons/icons/SortAscendingMajor.yml

```
@@ -12,5 +12,5 @@ authors:
   - Joe Thomas
 version: 1
```

I haven't been great at updating the versions but it's something I've started thinking about. We can probably do away with versioning since it's not surfaced anywhere but just in case let's update them

```suggestion
version: 2
```